test/e2e/remote/ssh.go

```diff
@@ -30,7 +30,8 @@ type Connection struct {
 func NewConnection(host, port, user, keyPath string) (*Connection, error) {
 	conn, err := net.Dial("unix", os.Getenv("SSH_AUTH_SOCK"))
 	if err != nil {
-		log.Fatal(err)
+		log.Println("unable to establish net connection, probably due to $SSH_AUTH_SOCK not being set")
```

Comment:
Here is it possible to add the value of SSH_AUTH_SOCK? Ie. ("unable to establish net connection, $SSH_AUTH_SOCK has value %s", os.Getenv("SSH_AUTH_SOCK"))